api/http/routing.go

```diff
@@ -58,11 +58,12 @@ const (
 	ApiUrlDevicesDeploymentStatus = ApiUrlDevices + "/device/deployments/:id/status"
 	ApiUrlDevicesDeploymentsLog   = ApiUrlDevices + "/device/deployments/:id/log"
 
-	ApiUrlInternalAlive             = ApiUrlInternal + "/alive"
-	ApiUrlInternalHealth            = ApiUrlInternal + "/health"
-	ApiUrlInternalTenants           = ApiUrlInternal + "/tenants"
-	ApiUrlInternalTenantDeployments = ApiUrlInternal + "/tenants/:tenant/deployments"
-	ApiUrlInternalTenantArtifacts   = ApiUrlInternal + "/tenants/:tenant/artifacts"
+	ApiUrlInternalAlive                          = ApiUrlInternal + "/alive"
+	ApiUrlInternalHealth                         = ApiUrlInternal + "/health"
+	ApiUrlInternalTenants                        = ApiUrlInternal + "/tenants"
+	ApiUrlInternalTenantDeployments              = ApiUrlInternal + "/tenants/:tenant/deployments"
+	ApiUrlInternalTenantArtifacts                = ApiUrlInternal + "/tenants/:tenant/artifacts"
+	ApiUrlInternalDeviceConfigurationDeployments = ApiUrlInternal + "/tenants/:tenant/devices/:id/configuration/deployments/:deployment_id"
```

Comment:
Given that we have `/tenants/:tenant/deployments`, it comes natural to me to imagine the path to be:
`/tenants/:tenant/configuration/deployments/:deployment_id/devices/:device_id`. What do you think?

Reply:
I was thinking that the deployment resource here is a subresource of the device (the device will have more than one configuration deployment), but it's the deployment, not the device deployment so you're probably right; I'll change it according to your suggestion

Reply:
done